src/ansys/fluent/core/session_base_meshing.py

```diff
@@ -184,6 +185,21 @@ def create_workflow(self):
             self.get_fluent_version(),
         )
 
+    @property
+    def current_workflow(self):
+        """Datamodel root of the workflow exposed in an object-oriented manner."""
```

Comment:
Given that this is a Python class, it ought to be redundant to say that this is _in an object-oriented manner_ - we should remove those comments.